docker/start.sh

```diff
@@ -5,10 +5,29 @@ if [ -z "$REINDEX" ]; then
 	REINDEX=10
 fi
 
+if [ "${OPENGROK_WEBAPP_CONTEXT}" = "/" ]; then
+	WAR_NAME="ROOT.war"
+else
+	WAR_NAME="${OPENGROK_WEBAPP_CONTEXT#/}.war"
```

Comment:
In Tomcat, one might specify a context path of e.g. `/source/opengrok` with an archive named `source#opengrok.war`. Might that be supported with substitution here?

Reply:
I'm unsure whether there are any advantages to that compared to simply filling in the URL-subfolder as-is (with minor changes to unify the format).

I'd also prefer the path-based version personally, since it's easier to handle and understand (in my opinion).

Reply:
My question should have been more general: what will happen if the user specifies `/source/opengrok` as a context?

Reply:
I just noticed that what I intended doesn't actually work. I thought that tomcat resolves the used .war file by the full path (i.e. that a path of `/usr/share/tomcat/webapp/source/opengrok.war` would result in OpenGrok being accessible at `/source/opengrok`, although `/source` is a whole other story), but that apparently only works for `ROOT` and first-level files.

I'll do some testing with substitutions and add a commit if everything seems fine.